developer_docs/pr_review.asciidoc

```diff
@@ -0,0 +1,53 @@
+= Reviewing Pull Requests
+:toc: right
+:toclevels: 5
+
+This document provide a guideance for the Github Pull Request review. It's not a complete list and can be little bit overhelming for smaller patches, however it sets the standard we should all aim for.
+
+[[How]]
+== How to use the list
+
+It's a good idea to use https://help.github.com/en/articles/using-saved-replies[Saved reply Github] feature and use it for patches which require extra attention.
+
+[[List]]
+== Review list
+
+- [ ] Ticket number, title and description correct?
+- [ ] Does it fix the problem described in the issue?
+- [ ] No unrelated changes?
+- [ ] Redmine opened and in correct project?
+- [ ] No use of .to_sym, .send or other reflection on untrusted inputs.
+- [ ] Not missing string extractions (use :mark_translated: true).
+- [ ] All string extractions follows our rules.
+- [ ] Added appropriate permissions for non-admin users?
+- [ ] New code hasn't been copy-pasted from elsewhere.
+- [ ] Do new fields have appropriate validators?
+- [ ] No exceptions are swaloved/squashed.
+- [ ] Are raised exceptions type of Foreman::Exception or WrappedException?
+- [ ] Unit/functional/integration test(s) present?
+- [ ] New view templates have .html.erb extensions, not just .erb
+- [ ] Mixins use ActiveSupport::Concern instead of class_eval etc.
+- [ ] Concerns and new classes are added to app/ and not lib/
+- [ ] Apipie documentation is correct: HTTP methods, names of parameters, required true/false
+- [ ] Scoped_search definitions are added for new model attributes where appropriate.
+- [ ] Scoped_search definitions for virtual fields (:ext_method) use :only_explicit
+- [ ] Makes use of logging appropriately?
+- [ ] Pass block to logger.debug to lazily evaluate expensive debug statements
+- [ ] Deprecations use Foreman::Deprecation with a deadline of latest stable + three versions.
+- [ ] Not changing structure or behavior of the public HTTP API?
+- [ ] No memory or performance concerns?
+- [ ] Any packaging needed?
+- [ ] User documentation needed?
+- [ ] Demo worthy?
+- [ ] Any other actions need to be taken after merge?
```

Comment:
I'd suggest adding
- [ ] needs to be added to API?
- [ ] needs hammer support?
- [ ] needs manual update?
- [ ] needs to be in release notes?

Reply:
Done.